internal/unionstore/memdb.go

```diff
@@ -166,10 +165,10 @@ func (db *MemDB) Reset() {
 	db.allocator.reset()
 }
 
-// SetDiskFullOpt is used by TiDB test case.
-func (db *MemDB) SetDiskFullOpt(level kvrpcpb.DiskFullOpt) {
-	// Nothing to do.
-}
+// // SetDiskFullOpt is used by TiDB test case.
+// func (db *MemDB) SetDiskFullOpt(level kvrpcpb.DiskFullOpt) {
+// 	// Nothing to do.
+// }
```

Comment:
Please remove the code.

Reply:
Right, tidb has already removed this func.
Done!